packages/overlay/src/vaadin-overlay.js

```diff
@@ -489,6 +493,7 @@ class Overlay extends ThemableMixin(DirMixin(ControllerMixin(PolymerElement))) {
       afterNextRender(this, () => {
         if (this.focusTrap) {
           this.__focusTrapController.trapFocus(this.$.overlay);
+          this.__ariaModalController.showModal();
```

Comment:
General question: does this consider the use-case where overlays are not modal and the content on the page is still accessible? Like e.g. a small dialog on the bottom that contains a chat where people can switch between the form in the "background" and the dialog that's always visible on the bottom right with the chat? (Archived by creating a dialog with modal = false and position)